### PR TITLE
Add link to release video

### PR DESCRIPTION
**Pull request by annajung**
Signed-off-by: Anna Jung (VMware) <antheaj@vmware.com>

- Add a link to the 1.6 release video

cc @kubeflow/release-team 

This will need to be cherry-picked to the v1.6-branch as well

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/lgtm

File: content/en/docs/releases/kubeflow-1.6.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ weight = 98
           <a href="https://blog.kubeflow.org/kubeflow-1.6-release/">Kubeflow 1.6 Release Announcement</a>
         <br>
         <b>Video:</b> 
-          TBA
+          <a href="https://youtu.be/RR1xSfnFGGI">Kubeflow 1.6 Release Overview</a>
```
Comment (thesuperzapper) [inline content/en/docs/releases/kubeflow-1.6.md:25]: Nit: can you use the expanded link: https://www.youtube.com/watch?v=RR1xSfnFGGI

File: content/en/docs/releases/kubeflow-1.6.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ weight = 98
...
         <br>
         <b>Roadmap:</b>
           <a href="https://github.com/kubeflow/kubeflow/blob/master/ROADMAP.md#kubeflow-16-release-due-september-2022">Kubeflow 1.6 Features</a>
```
Comment (thesuperzapper) [inline content/en/docs/releases/kubeflow-1.6.md:28]: Can you also update this link anchor to be correct (as we have changed the name of the heading), it should now be:

https://github.com/kubeflow/kubeflow/blob/master/ROADMAP.md#kubeflow-16-release-delivered-september-2022

**Review by thesuperzapper — CHANGES_REQUESTED** [review]
@annajung I will approve, but I have raised 2 very quick comments.

Also, I can't cherry-pick, but we could do a chery-pick PR (if someone who has write permission is not available).

File: content/en/docs/releases/kubeflow-1.6.md (hunk 1 above)
Comment (annajung) [reply content/en/docs/releases/kubeflow-1.6.md]: Fixed

**Pull request merged**